public/index.html

```diff
@@ -1,49 +1,86 @@
+         \/|__|       \/           \/         \/
 
-<body>
-  <noscript>You need to enable JavaScript to run this app.</noscript>
-  <div id="root"></div>
-  <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
+The Sparkle Online Event Platform is open-source! Come take a 👀!
 
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
+BUILD_SHA1           : %REACT_APP_BUILD_SHA1%
+BUILD_TAG            : %REACT_APP_BUILD_TAG%
+BUILD_BRANCH         : %REACT_APP_BUILD_BRANCH%
+BUILD_PULL_REQUESTS  : %REACT_APP_BUILD_PULL_REQUESTS%
+BUILD_REPOSITORY_URL : %REACT_APP_BUILD_REPOSITORY_URL%
 
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
-</body>
+- Made with ❤️️, powered by ✨
+-------------------------------------------------------------------
+-------------------------------------------------------------------
+*/
+    </script>
+    <meta name="generator" content="Sparkle Platform" />
+    <meta name="sparkle-build-sha1" content="%REACT_APP_BUILD_SHA1%" />
+    <meta name="sparkle-build-tag" content="%REACT_APP_BUILD_TAG%" />
+    <meta name="sparkle-build-branch" content="%REACT_APP_BUILD_BRANCH%" />
+    <meta
+      name="sparkle-build-pull-requests"
+      content="%REACT_APP_BUILD_PULL_REQUESTS%"
+    />
+    <meta
+      name="sparkle-build-repository"
+      content="%REACT_APP_BUILD_REPOSITORY_URL%"
+    />
+    <meta property="al:web:url" content="%REACT_APP_BUILD_REPOSITORY_URL%" />
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="%PUBLIC_URL%/favicon/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="%PUBLIC_URL%/favicon/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="%PUBLIC_URL%/favicon/favicon-16x16.png"
+    />
+    <meta name="msapplication-TileColor" content="#ffffff" />
+    <meta name="theme-color" content="#000000" />
+    <meta name="description" content="Sparkle" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <title>Sparkle</title>
+    <script src="https://www.eventbrite.co.uk/static/widgets/eb_widgets.js"></script>
+  </head>
 
+  <body
+    data-sparkle-build-sha1="%REACT_APP_BUILD_SHA1%"
+    data-sparkle-build-tag="%REACT_APP_BUILD_TAG%"
+    data-sparkle-build-branch="%REACT_APP_BUILD_BRANCH%"
+    data-sparkle-build-pull-requests="%REACT_APP_BUILD_PULL_REQUESTS%"
+    data-sparkle-build-repository="%REACT_APP_BUILD_REPOSITORY_URL%"
```

Comment:
This seems like a little too much info, no need to include it in our page just because it's there, I'd advocate a more minimal approach. What about we only give the sha and repo URL. The rest can be derived from them anyway.

Reply:
These were explicit choices narrowed down from what was available. I don't feel strongly about keeping it in the `data-` keys (as it is already in `meta`), but unless there is a strong reason not to, I feel pretty strongly about keeping the information here in the index in general.

- It 'costs' us basically nothing, a few extra characters in the index, trivial compared to the size of the rest of the app's assets.
- We already use this data to pass extra context on Bugsnag errors, so it needs to exist within the app regardless of whether it is exposed here.
- This is part of my plan to 'entice curious developers to look deeper/explore'. It's easy for us to infer/trace back based on a minimal set of information. I want to lower that barrier of required knowledge to do so. If the information is simple and easy to follow, there's a higher chance of follow-through based on it.

Open to hearing your thoughts/reasons why we shouldn't.

Reply:
Couple of thoughts / responses, but I'm not going to push either way as it's a minor change and you're the one leading it so you should make the call.

* Agreed on zero cost. With caching, compression etc. it's even less.

* Re Bugsnag. My thinking was about reducing redundancy in what we surface to humans; I'm a minimalist - and the rest can be derived. It depends what impression we want to give; I feel as though showing all the extra info triggers OCD that perhaps "these people haven't whittled down the things they include to the bare minimum". I like the saying, attributed to Antoine de Saint-Exupéry:

> Perfection is achieved, not when there is nothing more to add, but when there is nothing left to take away.

* Re enticing developers - yes this info is required to initialize Bugsnag but it's not required (in my opinion) to get humans interested in digging further. All it takes to send me spelunking is a sha and a url.

Reply:
I hear you on reducing redundancy in general, I also like a clean/minimalist approach to things, and for me, a repo URL/SHA (or even less ;) ) is all it would take me to track down a repo and get involved. Though my thinking with adding this little bit extra here is less for the hardcore/elite, and more to give an extra helping hand to someone who might be interested in a curiously dabbling/exploring the project, and reducing that barrier of entry/discoverability.

My original reason for adding the `data-` tags as well was to increase potential discoverability (as the `index.html` gets quite aggressively minimized during our build process), but reflecting, since I worked out how to keep an uncompressed comment, + it existing in `meta`, I don't think it adds anything extra we don't already have. So I'll delete these `data-` keys, and then will probably merge this PR.

If we find it too overbearing in future, we can always address things in a followup PR then. (eg. right now i'm not familiar enough to know how much extra context passed to bugsnag is useful, I expect we may be able to pare that down a bit once we get some data/real-world usage experience with it, and which bits we use vs which we just kind of ignore/feel redundant.)

Reply:
Just thinking some more. Removing it entirely might not entice people To be clear, here's what I was advocating we do:

```
    data-sparkle-build-sha1="%REACT_APP_BUILD_SHA1%"
    data-sparkle-build-repository="%REACT_APP_BUILD_REPOSITORY_URL%"
```

That'd get me going "ooo I wonder what's in that sha!" - thoughts on doing that?

Reply:
The `data-` keys get pretty lost in the minified html anyway, so they're not super discoverable anyway.

Easier to describe visually:
![Screen Shot 2020-12-02 at 11 18 14 am](https://user-images.githubusercontent.com/753891/100812173-8074b800-3490-11eb-9b32-8a801d950356.png)

I'm keeping 1 and 2, removed 3 in https://github.com/sparkletown/sparkle/pull/847/commits/abf7d7e9f7e5407431cee02f0f1bb7c0ac1ddfa3

Reply:
Thanks for the visual. Looks good.